src/patterns/data/default/artery_and_arteriole_pattern.tsv

```diff
@@ -18,4 +18,6 @@ UBERON:8600098	dorsal lingual artery	FMA:49546	UBERON:0001637	UBERON:0001456	"An
 UBERON:8600101	descending palatine artery	FMA:49791	UBERON:0001637	UBERON:0001456	An artery that branches from the maxillary artery and supplies the hard and soft palate.	wikipedia:Descending_palatine_artery			UBERON:0001616	UBERON:0003216|UBERON:0001733	https://orcid.org/0000-0001-6677-8489	2024-06-06T09:53:48Z
-UBERON:8600102	left anterior descending artery	FMA:3862	UBERON:0001637	UBERON:0000915	"An artery that branches from the left coronary artery and supplies the anterior wall of the left ventricle, the interventricular septum, and the apex of the heart."	wikipedia:Left_anterior_descending_artery	anterior interventricular branch of left coronary artery	FMA:3862	UBERON:0001626		https://orcid.org/0000-0001-6677-8489	2024-06-06T09:53:48Z
\ No newline at end of file
+UBERON:8600102	left anterior descending artery	FMA:3862	UBERON:0001637	UBERON:0000915	"An artery that branches from the left coronary artery and supplies the anterior wall of the left ventricle, the interventricular septum, and the apex of the heart."	wikipedia:Left_anterior_descending_artery	anterior interventricular branch of left coronary artery	FMA:3862	UBERON:0001626		https://orcid.org/0000-0001-6677-8489	2024-06-06T09:53:48Z
+UBERON:8600106	anterior humeral circumflex artery	FMA:22680	UBERON:0001637	UBERON:0001467	"An artery that branches from the axilliary artery and supplies the deltoid, biceps brachii, and coracobrachialis muscles, the glenohumeral joint, the proximal humeral head, and the skin overlying the anterior aspect of the proximal humerus. This artery courses laterally across the anterior surface of the humerus and anastomoses with the posterior humeral circumflex artery."	https://www.elsevier.com/resources/anatomy/cardiovascular-system/arteries/anterior-circumflex-humeral-artery/20091			UBERON:0001394	UBERON:0001476|UBERON:0001507|UBERON:0001505|UBERON:0001470|UBERON:0006801	https://orcid.org/0000-0001-6677-8489	2024-08-20T09:53:48Z
+UBERON:8600107	posterior humeral circumflex artery	FMA:22684	UBERON:0001637	UBERON:0001467	"An artery that branches from the axilliary artery and supplies the deltoid, teres major, teres minor, and long and lateral heads of the triceps brachii muscles, glenohumeral joint, and the cutaneous supply to the area posterolateral to the proximal humerus. This artery courses posteriorly with the axillary nerve, through the quadrangular space, to the posterior aspect of the humerus and anastomoses with the anterior humeral circumflex artery."	https://www.elsevier.com/resources/anatomy/cardiovascular-system/arteries/posterior-circumflex-humeral-artery/24676			UBERON:0001394	UBERON:0001476|UBERON:0001478|UBERON:0010496|UBERON:0001509|UBERON:0001470	https://orcid.org/0000-0001-6677-8489	2024-08-20T09:53:48Z
```

Comment:
You say in the definition: "supplies ... the cutaneous supply to the area posterolateral to the proximal humerus".
Does that mean that the list `vessel_supplies_blood_to` should also include the posterolateral skin of the shoulder, or the closest term in Uberon, [upper arm skin](http://purl.obolibrary.org/obo/UBERON_0004263)?

You might want to add `arteria circumflexa posterior humeri` as a synonym (`synonym_xrefs`: `https://ta2viewer.openanatomy.org/?id=4631`).

Reply:
Thank you for your comment, same answer as in the previous comment.